do-catalog/src/views/DatasetSummary.vue

```diff
@@ -127,8 +127,8 @@ export default {
       return geometryTypeName(this.dataset.geom_type);
     },
     spatialAggregation() {
-      return this.dataset.spatial_aggregation
-        ? toTitleCase(this.dataset.spatial_aggregation)
+      return this.dataset.spatial_aggregation_name
```

Comment:
I think @antoniocarlon commented that this must be `spatial_aggregation`. So let's change it and :rocket:

Reply:
Nope, this must be `spatial_aggregation_name` (`spatial_aggregation` was the old name)

Reply:
But this (the Dataset detail) is obtained from the /dataset/:id or /geography/:id endpoints. Do these endpoints include `spatial_aggregation_name` instead of `spatial_aggregation`?

Reply:
Neither the `/dataset/:id` nor the `/geography/:id` endpoints return the spatial aggregation

Reply:
OK